src/jit/codegenarm64.cpp

```diff
@@ -5453,7 +5452,7 @@ void CodeGen::genJmpMethod(GenTreePtr jmp)
 
         // Is register argument already in the right register?
         // If not load it from its stack location.
-        var_types  loadType  = varDsc->lvaArgType();
+        var_types loadType = compiler->mangleVarArgsType(genActualType(varDsc->TypeGet()));
```

Comment:
Not sure whether we have "jmp method" uni tests that pass a struct in two regs.  You might want to add one to test it.